src/engine/engineworkerscheduler.cpp

```diff
@@ -43,14 +44,14 @@ void EngineWorkerScheduler::run() {
     while (!m_bQuit) {
         Event::start(tag);
         {
-            QMutexLocker lock(&m_mutex);
+            const auto locker = lockMutex(&m_mutex);
```

Comment:
Why did you rename this occurrence manually?

For the future I suggest to use `const auto lockedMutex = lockMutex(...);` instead, but we don't need to do this in this PR.

Reply:
copying and pasting? :shrug:

Reply:
It looked like you used search&replace as a batch operation, but not for this occurrence. I was just wondering why.